DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/DiagnosticContentViewModelTests.cs

```diff
@@ -0,0 +1,240 @@
+
+            // When
+            var diagnosticContentViewModel = new DiagnosticContentViewModel(
+                config,
+                diagnosticContent,
+                SelectedTutorials,
+                customisationId,
+                CentreId,
+                SectionId,
+                ProgressId,
+                CandidateId
+            );
+
+            // Then
+            diagnosticContentViewModel.CustomisationId.Should().Be(customisationId);
+        }
+
+        [Test]
+        public void Diagnostic_content_should_have_section_id()
+        {
+            // Given
+            const int sectionId = 22;
+            var diagnosticContent = DiagnosticAssessmentHelper.CreateDefaultDiagnosticContent();
+
+            // When
+            var diagnosticContentViewModel = new DiagnosticContentViewModel(
+                config,
+                diagnosticContent,
+                SelectedTutorials,
+                CustomisationId,
+                CentreId,
+                sectionId,
+                ProgressId,
+                CandidateId
+            );
+
+            // Then
+            diagnosticContentViewModel.SectionId.Should().Be(sectionId);
+        }
+
+        [Test]
+        public void Diagnostic_content_should_parse_html_url()
+        {
+            // Given
+            const int centreId = 11;
+            const int customisationId = 22;
+            const int candidateId = 33;
+            const int sectionId = 44;
+            const int currentVersion = 55;
+            const int progressId = 66;
+            const int plaPassThreshold = 77;
+            const string diagAssessPath = "https://www.dls.nhs.uk/tracking/RFM/L1_Word10/Assess/L1_2.03_Diag.dcr";
+            var selectedTutorials = new List<int>(new[] { 1, 2, 3 });
+            var diagnosticContent = DiagnosticAssessmentHelper.CreateDefaultDiagnosticContent(
+                diagAssessPath: diagAssessPath,
+                plaPassThreshold: plaPassThreshold,
+                currentVersion: currentVersion
+            );
+
+            // When
+            var diagnosticContentViewModel = new DiagnosticContentViewModel(
+                config,
+                diagnosticContent,
+                selectedTutorials,
+                customisationId,
+                centreId,
+                sectionId,
+                progressId,
+                candidateId
+            );
+
+            // Then
+            diagnosticContentViewModel.ContentSource.Should().Be(
+                "https://www.dls.nhs.uk/tracking/RFM/L1_Word10/Assess/L1_2.03_Diag.dcr" +
+                "?CentreID=11&CustomisationID=22&CandidateID=33&SectionID=44&Version=55&ProgressID=66" +
+                $"&type=diag&TrackURL={BaseUrl}/tracking/tracker&objlist=[1,2,3]&plathresh=77"
+            );
+        }
+
+        [Test]
+        public void Diagnostic_content_should_have_tutorials()
```

Comment:
This test doesn't match the name. In fact it looks like a copy of the test above? 🙂